dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -414,6 +414,7 @@ yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.stripe.teamUsagePriceIds[
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.configcatKey "WBLaCPtkjkqKHlHedziE9g/LEAOCNkbuUKiqUZAcVg7dw"
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.proxy.configcat.baseUrl "https://cdn-global.configcat.com"
 yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.proxy.configcat.pollInterval "1m"
+yq w -i "${INSTALLER_CONFIG_PATH}" experimental.webapp.proxy.configcat.fromConfigMap "configcat"
```

Comment:
This will effectively disable feature flags in preview, right? 

What is responsible for updating the configmap with the latest values? Does that component already exist?